app/Filament/Subcontractor/Widgets/SubcontractorDashboardWidget.php

```
@@ -0,0 +1,111 @@
+            ->where('status', ChantierStatus::IN_PROGRESS)
+            ->count();
+
+        $pendingInvoices = SubcontractorSituation::where('subcontractor_id', $thirdParty->id)
+            ->whereIn('status', [InvoiceStatus::DRAFT, InvoiceStatus::VALIDATED])
+            ->count();
+
+        $pendingAmount = SubcontractorSituation::where('subcontractor_id', $thirdParty->id)
+            ->whereIn('status', [InvoiceStatus::DRAFT, InvoiceStatus::VALIDATED])
+            ->sum('total_ht');
+
+        $upcomingTasks = ChantierTask::whereHas('allocations', function ($query) use ($thirdParty) {
```

PHPStan (2.2.7)

[error] Call to an undefined static method App\Models\Chantiers\ChantierTask::whereHas().


(staticMethod.notFound)